packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -483,30 +478,142 @@ export class PerformanceEvent {
     }
 }
 
+// Note - these Telemetry types should move to common-definitions package
+
+/**
+ * Broad classifications to be applied to individual properties as they're prepared to be logged to telemetry.
+ * Please do not modify existing entries for backwards compatibility.
+ */
+export enum TelemetryDataTag {
```

Comment:
i think these should be give explict values like `None=0` may even want strings if the will be logged, `None="None`